instance/models/instance.py

```diff
@@ -119,11 +123,13 @@ class Instance(ValidateModelMixin, models.Model):
     # in a query, e.g. to do .select_related('ref_set')
     ref_set = GenericRelation(InstanceReference, content_type_field='instance_type', object_id_field='instance_id')
     openstack_region = models.CharField(
+        # TODO: Move this to OpenedXInstance
```

Comment:
I think some larger refactoring is in order. e.g. Move server creation elsewhere (it must be made more generic anyways in the future to support multiple OpenStack accounts and/or non-OpenStack clouds like AWS/GCP).

In any case, even with the current design it could be possible for the `Instance` class to do something like `create_server(..., **self.get_subclass_instance().get_server_type_arguments())`